signup_success.php

```diff
@@ -1,3 +1,3 @@
 <h2>Signup Successful</h2>
 
-<p>Your Account Has Been Successfully Created. You Can Now <a href="index.php?sid=login">Log in</a>.</p>
\ No newline at end of file
+<p>Your Account Has Been Successfully Created. You Can Now <a href="index.php?sid=login">Log in</a>.</p>
```

Comment:
Můžeš použít i automatický přesměrovač po x sekundách např. :slightly_smiling_face: 

```suggestion
<p>Your Account Has Been Successfully Created. You Can Now <a href="index.php?sid=login">Log in</a>.</p>
<?php header("refresh:5;url=index.php?sid=login") ?>
```